README.md

```diff
@@ -95,6 +95,34 @@ For example:
 install-php-extensions mongodb-stable
 ```
 
+### Installing from source code
+
+You can also install PHP extensions from source code (provided that it comes with a `package.xml` or a `package2.xml` file).
+
+Accepted formats are:
+
+- A short version for GitHub.
+  For example, to install from commit `8f106564e6bb005ca6100b12ccc89000daafa9d8` of the [php-memcached-dev/php-memcached](https://github.com/php-memcached-dev/php-memcached) GitHub repository,
+  you can simply write:
+  ```sh
+  install-php-extensions php-memcached-dev/php-memcached@8f106564e6bb005ca6100b12ccc89000daafa9d8
```

Comment:
very nice, can this short format for Github be used also with branch/tag name? if so, I belive this should be also the recommended format beside install from tar.gz link or local path

Reply:
See 6045b535638e716a82de18b6475fc47c81572425